### PR TITLE
A job to send reminder on the opened PR for gerrit migration

### PR DESCRIPTION
**Pull request by deepshikhaaa**

### CODE REVIEW

File: build-gluster-org/scripts/comment-on-open-pr.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    cleaned_output = '\n'.join(output.split('\n')[1:])
+    parsed_output = json.loads(cleaned_output)[1:]
+
+
+for change in parsed_output:
+        url = ('https://review.gluster.org/a/changes/{}/revisions/current/review'
+               .format(change['id']))
+        data = {
+            "message": "This PR needs to be migrated to Github glusterfs repo"
+                       "More information about migration can be forund here:"
```
Comment (mscherer) [inline build-gluster-org/scripts/comment-on-open-pr.py:24]: Typo (forund)

File: build-gluster-org/scripts/comment-on-open-pr.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import os
+import sys
+import requests
+
+r = requests.get('https://review.gluster.org/changes/'
+                     '?q=status:open+'
+                     'project:glusterfs')
+
+output = r.text
+    cleaned_output = '\n'.join(output.split('\n')[1:])
```
Comment (mscherer) [inline build-gluster-org/scripts/comment-on-open-pr.py:15]: I am a bit surprised, the indentation seems not correct.

**Review by mscherer — COMMENTED** [review]
Seems good, minus my question

File: build-gluster-org/scripts/comment-on-open-pr.py (hunk 2 above)
Comment (deepshikhaaa) [reply build-gluster-org/scripts/comment-on-open-pr.py]: Ah, I thought I fixed it and already pushed with the changes!

**Comment by mscherer** [comment]
Look good to me

**Pull request merged**